WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/PointOfSaleItemListErrorView.swift

```diff
@@ -58,7 +68,31 @@ struct PointOfSaleItemListErrorView: View {
     }
 }
 
+struct PointOfSaleItemListErrorViewModel {
+    let title: String
+    let subtitle: String
+    let buttonText: String
+    let imageName: String?
```

Comment:
WDYT about using the `PointOfSaleAssets` case here directly, rather than a string? Perhaps it makes the viewmodel clumsier when it's used, but at least it helps to avoid people using untested strings in future...